build/build.sh

```diff
@@ -49,6 +49,11 @@ if [ -n "$VERBOSE" ]; then
   echo "Building with -ldflags $ldflags"
 fi
 
-GOBIN=$PWD go build ${GO_FLAGS} -ldflags "${ldflags}" "${repo_path}"
+if [ $1 == "ppc64le" ]
+then
+  GOBIN=$PWD env GOOS=linux GOARCH=ppc64le go build ${GO_FLAGS} -ldflags "${ldflags}" "${repo_path}"
```

Comment:
This can be easily made into any-arch build script by first assigning `GOARCH=$1` and then inverting condition to test if `GOARCH` is not empty.

Reply:
Thanks!
Have made necessary changes to the conditional.